server/src/main/java/io/whitefox/persistence/memory/PTable.java

```diff
@@ -1,3 +1,3 @@
 package io.whitefox.persistence.memory;
 
-public record PTable(String name) {}
+public record PTable(String name, String location) {}
```

Comment:
Do we want to perform some validation on the location?

Reply:
it does not seem necessary for delta sharing APIs, since table model for delta sharing does not include location
it can be validated on storage manager tests? since it is part of 'our' class

Reply:
sounds good to me